### ISSUE TITLE
Please create a bower package

### ISSUE THREAD

**Issue by JosephShering** [GriddleGriddle/Griddle#82]
That would be swell for some of us not using npm.

**Comment by danawoodman** [comment]
@JosephShering did you try and install it with bower (`bower install DynamicTyped/Griddle`) and then link to the file `build/griddle.js`?

**Comment by JosephShering** [comment]
Yeah, I get: 

```
bower not-cached    git://github.com/DynamicTyped/Griddle.git#*
bower resolve       git://github.com/DynamicTyped/Griddle.git#*
bower download      https://github.com/DynamicTyped/Griddle/archive/v0.2.0.tar.gz
bower extract       Griddle#* archive.tar.gz
bower invalid-meta  Griddle is missing "main" entry in bower.json
bower invalid-meta  Griddle is missing "ignore" entry in bower.json
bower resolved      git://github.com/DynamicTyped/Griddle.git#0.2.0
bower install       Griddle#0.2.0
```

**Comment by ryanlanciaux** [comment]
Thanks for the suggestion. We will add to bower soon :+1:

**Comment by danawoodman** [comment]
@JosephShering Weird... it's working for me, anything specific about your environment?

**Comment by emombay** [comment]
Any way we could get an update of the bower package? It's currently pointing to v0.2.15

**Comment by emombay** [comment]
Not sure that worked... this is what I get after clearing bower cache:
```
bower griddle-react#>=0     not-cached git://github.com/GriddleGriddle/Griddle.git#>=0
bower griddle-react#>=0        resolve git://github.com/GriddleGriddle/Griddle.git#>=0
bower griddle-react#>=0       download https://github.com/GriddleGriddle/Griddle/archive/v0.2.15.tar.gz
bower griddle-react#>=0        extract archive.tar.gz
bower griddle-react#>=0       mismatch Version declared in the json (0.2.13) is different than the resolved one (0.2.15)
bower griddle-react#>=0       resolved git://github.com/GriddleGriddle/Griddle.git#0.2.15
bower griddle-react#>=0        install griddle-react#0.2.15

griddle-react#0.2.15 bower_components\griddle-react
├── react#0.14.7
└── underscore#1.8.3
```
I do have the latest bower version.

**Comment by hagabaka** [comment]
@ryanlanciaux needs to add the v0.4.1 git tag, using the commands I included in #373. I can't do that in a pull request.

**Comment by ryanlanciaux** [comment]
Hi everyone - I think we're going to close this one for now. Please feel free to comment if you're encountering this trouble still and sorry about the extreme delay on this one!
